### PR TITLE
Load MySQL comments into TableSchema

### PR DESCRIPTION
**Pull request by michaelarnauts**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes/no
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #13154 (partially)

### CODE REVIEW

File: framework/db/TableSchema.php [hunk 1]
```diff
@@ -35,6 +35,10 @@ class TableSchema extends Object
      */
     public $fullName;
     /**
+     * @var string the comment that has been assigned to this table.
```
Comment (samdark) [inline framework/db/TableSchema.php]: Need `@since 2.0.12`.

File: framework/db/mysql/Schema.php [hunk 2]
```diff
@@ -122,6 +123,27 @@ protected function resolveTableNames($table, $name)
     }
 
     /**
+     * Loads the table comment into the [[TableSchema]] object.
+     * @param TableSchema $table the table metadata object
```
Comment (samdark) [inline framework/db/mysql/Schema.php]: Need `@since 2.0.12`.

Comment (samdark) [inline framework/db/mysql/Schema.php]: How about other database engines?

**Review by samdark — CHANGES_REQUESTED** [review]

**Comment by michaelarnauts** [comment]
I've added the `@since 2.0.12` to the phpdoc.

**Comment by samdark** [comment]
Great. We can't really release it for MySQL only so will see if we'll have enough time to implement all other DB engines support before 2.0.12...

**Comment by michaelarnauts** [comment]
* `mssql` uses a strange way to store comments, by using "extended properties", not sure if this is considered a standard practice to do so (http://stackoverflow.com/questions/378700/is-it-possible-to-add-a-description-comment-to-a-table-in-microsoft-sql-2000)
* `postgres` seems to provide a decent way to do this (http://stackoverflow.com/questions/5664094/getting-list-of-table-comments-in-postgresql)

I'm not sure about other databases.

**Comment by samdark** [comment]
Oracle: http://stackoverflow.com/questions/16567129/how-to-get-table-comments-via-sql-in-oracle

**Comment by samdark** [comment]
SQLite seems not to support comments at all.

**Comment by samdark** [comment]
Won't make it into 2.0.

**Comment by samdark** [comment]
https://github.com/yiisoft/yii2/pull/13162

**Pull request closed without merging**